dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackerOwnershipManagerTest.java

```diff
@@ -537,6 +537,22 @@ void shouldNotFindTrackedEntityWhenProgramSuppliedAndUserIsNotOwner()
     assertIsEmpty(getTrackedEntities(operationParams));
   }
 
+  @Test
+  void shouldFindTrackedEntityWhenCaptureScopeIndependentFromSearchScope()
+      throws ForbiddenException, NotFoundException, BadRequestException {
+    programA.setAccessLevel(AccessLevel.OPEN);
+    programService.updateProgram(programA);
+    userB.setOrganisationUnits(Set.of(organisationUnitA));
+    userB.setTeiSearchOrganisationUnits(Set.of(organisationUnitB));
+    userService.updateUser(userB);
+    injectSecurityContext(UserDetails.fromUser(userB));
+
+    assertEquals(
+        trackedEntityA1,
+        trackedEntityService.getTrackedEntity(
```

Comment:
Why is this in the TrackerOwnershipManagerTest? I think this should be part of our TE service test